src/main/java/org/prebid/server/handler/VersionHandler.java

```diff
@@ -24,43 +18,25 @@
 
     private static final Logger logger = LoggerFactory.getLogger(VersionHandler.class);
 
-    private static final String UNDEFINED = "undefined";
-
     private final String revisionResponseBody;
 
-    private VersionHandler(String revisionResponseBody) {
+    public VersionHandler(String revisionResponseBody) {
         this.revisionResponseBody = revisionResponseBody;
     }
 
-    public static VersionHandler create(String revisionFilePath, JacksonMapper mapper) {
-        Revision revision;
-        try {
-            revision = mapper.mapper().readValue(ResourceUtil.readFromClasspath(revisionFilePath), Revision.class);
-        } catch (IllegalArgumentException | IOException e) {
-            logger.error("Was not able to read revision file {0}. Reason: {1}", revisionFilePath, e.getMessage());
-            revision = Revision.EMPTY;
-        }
-        return new VersionHandler(createRevisionResponseBody(revision, mapper));
+    public static VersionHandler create(String version, String commitHash, JacksonMapper mapper) {
```

Comment:
Do we really need helper method to the handler?
Maybe for simplicity we can use plain constructor since no exceptions throwing.